assets/chat/css/weather.scss

```diff
@@ -0,0 +1,95 @@
+
+/*
+ * Minecraft weather effects with pure CSS
+ * © Copyright 2014 Moritz Marquardt
+ *
+ * https://github.com/moqmar/weather.css
+ *
+ * Licensed under MIT License (http://opensource.org/licenses/MIT)
+ */
+
+/* main element */
+.weather {
+    position: relative; 
+    overflow: hidden;
+    padding-top: 10px;
+    margin-top:-10px;
+    margin-bottom: -12px;
+}
+
+.weather > .chat-emote {
+    margin-bottom: 4px;
+}
+
+/* pseudo elements: positioning and setup */
+.weather:before, .weather:after {
+    content: "";
+    position: absolute; left: -50%; top: -50%; right: -50%; bottom: -50%;
+    z-index: 1;
+    pointer-events: none;
+    
+    background: transparent repeat;
+    background-size: 256px 1024px;
+    -ms-interpolation-mode: nearest-neighbor;
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: -webkit-crisp-edges;
+    image-rendering: -moz-crisp-edges;
+    image-rendering: -o-crisp-edges;
+    image-rendering: pixelated;
```

Comment:
Is it necessary to include these browser vendor properties? Don't we have autoprefixer? @KenanY

Reply:
This file is from a dependency.

Reply:
what dependency

Reply:
<https://github.com/moqmar/weather.css>